worker/lib/workers/testbot.ts

```diff
@@ -147,6 +147,9 @@ class TestBotWorker extends EventEmitter implements Leviathan.Worker {
 			if (this.networkCtl != null) {
 				await this.networkCtl.teardown();
 			}
+
+      unlink(dutSerialPath);
```

Comment:
@vipulgupta2048 I noticed that the local serial log file was super long... we need to do something like this to clean it up between runs I guess

Reply:
Yo, I was just going to ask you this. How long (size-wise) are we talking about for the logs? Also, regarding "between runs" do you mean like between suites? or between 1 Hup to another

Reply:
@vipulgupta2048 between client runs, I guess it just concat's the DUT serial logs as long as the worker container stays around

Reply:
That seems about right. I am checking serial logs for a multi-suite that happens on the same device and there's subtle differences at the start. I was checking this https://jenkins.dev.resin.io/job/leviathan-raspberrypi4-64/856/ LGTM to have this

Reply:
@rcooke-warwick What do you think